skins/cat17/src/scripts/donationForm.js

```diff
@@ -138,68 +138,51 @@ $( function () {
 			stateKey: 'donationFormContent.addressType'
 		},
       {
-        viewHandler: WMDE.View.createFieldValueValidityIndicator( $( '#first-name' ) ),
+        viewHandler: WMDE.View.createFieldValueValidityIndicator( $( '.field-firstname' ) ),
```

Comment:
I was very excited to see the ids replaced by class selectors - making half the view handlers redundant in the process. OTOH is adds a certain amount of redundancy (separate attributes only used by one view handler/component each) for now. Would you agree that the general direction, step by step, is to reduce this back to one unique way to identify each input, maybe even reuse elements selected once in the main dispatchers (à la page contract in e.g. `donationForm.js`)

Reply:
Yes, that's a direction I'd like to take. Thanks for pointing it out.